source/usage-examples/command.txt

```diff
@@ -0,0 +1,46 @@
+=============
+Run a Command
+=============
+
+.. default-domain:: mongodb
+
+You can run all raw database operations using the
+``MongoDatabase.runCommand()`` method. A raw database operation is a
+command you can execute directly on the MongoDB server CLI. Commands
+include administrative and diagnostic tasks, such as fetching server
+stats or initializing a replica set. Call the ``runCommand()`` method
+with a Bson command object on an instance of a database to run your raw
+database operation.
+
+.. note::
+    Use the :manual:`mongo shell </reference/mongo-shell>` for
+    administrative tasks instead of the Java driver whenever possible.
+
+The ``runCommand()`` method accepts a command in Bson format.
+By default, ``runCommand`` returns an object of type
+``org.bson.Document`` containing the output of the database command. You
+can specify a return type for ``runCommand()`` as an optional second
+parameter.
+
+Example
+-------
+
```

Comment:
I think it would be useful to add a description of the code snippet here:

In the following sample code, we send the ``dbStats`` command to request statistics of a specific MongoDB database.

(and include dbStats in the list of links at the end)